### PR TITLE
Elrond initial implementation

### PR DESCRIPTION
**Pull request by miiu96**
Elrond Proxy URL : https://api.elrond.com

In order to integrate Elrond in blockatlas we have used the following API calls against our proxy:

    /network/status/4294967295: gets the latest block nonce (number) - for method CurrentBlockNumber()
    /block/4294967295/:nonce: gets a block by nonce (number) - for method GetBlockByNumber()
    /address/:accountAddress/transactions: gets latest transactions of a given account - for method GetTxsOfAddress()

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ The observer API watches the chain for new transactions and generates notificati
 <a href="https://cosmos.network/" target="_blank"><img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/cosmos/info/logo.png" width="32" /></a>
 <a href="https://bitcoin.org/" target="_blank"><img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/bitcoin/info/logo.png" width="32" /></a>
 <a href="https://harmony.one/" target="_blank"><img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/harmony/info/logo.png" width="32" /></a>
+<a href="https://elrond.com/" target="_blank"><img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/elrond/info/logo.png" width="32" /></a>
```
Comment (catenocrypt) [inline README.md:39]: This link is error 404; elrong blockchain is not supported by assets.

**Comment by catenocrypt** [comment]
Diff coverage is 35%, should be increased.

File: README.md (hunk 1 above)
Comment (miiu96) [reply README.md:39]: Indeed, we did not open a pull request against the repository `trustwallet/assets` yet

File: platform/elrond/client.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		ID:     block.Hash,
+		Txs:    txs,
+	}, nil
+}
+
+func (c *Client) GetTxsOfAddress(address string) (blockatlas.TxPage, error) {
+	var txPage TransactionsPage
+	// TODO: enable pagination of Elrond transactions in the future.
+	// TODO: currently Elrond only fetches the most recent 20 transactions.
+	path := fmt.Sprintf("address/%s/transactions", address)
```
Comment (EnoRage) [inline platform/elrond/client.go:49]: We fetch only latest 25 transactions

**Comment by EnoRage** [comment]
Great work!

**Pull request merged**